api/src/main/java/org/datacleaner/descriptors/DescriptorProvider.java

```diff
@@ -46,12 +46,6 @@
 public interface DescriptorProvider {
 
     /**
-     * @return a collection of {@link DescriptorProviderStatus} describing
-     *         current status.
-     */
-    public Collection<DescriptorProviderStatus> getStatus();
-
-    /**
```

Comment:
This pull request effectively removes all usages of the `DescriptorProviderStatus` interface, why not remove that interface as well?